lib/events/events.go

```diff
@@ -199,13 +204,20 @@ func UnmarshalEventType(s string) EventType {
 
 const BufferSize = 64
 
-type Logger struct {
-	subs                []*Subscription
+type Logger interface {
+	suture.Service
+	Log(t EventType, data interface{})
+	Subscribe(mask EventType) Subscription
+}
+
+type evLogger struct {
```

Comment:
Since we are in package events, I think this should be just `type logger`?

Reply:
That conflicts with the `logger` package. I could alternatively import that under a different name, not sure what would be appropriate.

Reply:
The logger package is only referred to in debug.go so I don't think that's a problem.

(Bonus idea to seed an idea for a future PR: how about merging the logger and the event system, having only events but having some of them formatted for log output on stdout?!)

Reply:
Go is too strict for that:

```
lib/events/debug.go:13:2: logger redeclared in this block
        previous declaration at lib/events/events.go:213:6
```

I'll use `liblogger` for the package.

(Bonus reaction: \*lalalalalala\* mustn't listen (for now (...))

Reply:
Interesting. But in that case yes, I'd import it with another name as it's just in the debug.go